docs/sources/project/set-up-prereqs.md

```diff
@@ -97,202 +110,209 @@ To fork and clone Docker:
+You must sign with your real name. We don't accept anonymous contributions or
+contributions through pseudonyms.
 
-As you change code in your fork, you'll want to keep it in sync with the changes others make in the `docker/docker` repository.  To make syncing easier, you'll also add a _remote_ called `upstream` that points to `docker/docker`. A remote is just another a project version hosted on the internet or network.
+As you change code in your fork, you'll want to keep it in sync with the changes
+others make in the `docker/docker` repository.  To make syncing easier, you'll
+also add a _remote_ called `upstream` that points to `docker/docker`. A remote
+is just another a project version hosted on the internet or network.
 
 To configure your username, email, and add a remote:
-		
+
 1. Change to the root of your `docker-fork` repository.
 
-		$ cd docker-fork
+        $ cd docker-fork
+
+2. Set your `user.name` for the repository.
 
-2. 	Set your `user.name` for the repository.
+        $ git config --local user.name "FirstName LastName"
 
-		$ git config --local user.name "FirstName LastName"
-			
 3. Set your `user.email` for the repository.
 
-		$ git config --local user.email "emailname@mycompany.com"
-	
+        $ git config --local user.email "emailname@mycompany.com"
+
 4. Set your local repo to track changes upstream, on the `docker` repository. 
 
-		$ git remote add upstream https://github.com/docker/docker.git
+        $ git remote add upstream https://github.com/docker/docker.git
 
 7. Check the result in your `git` configuration.
 
-		$ git config --local -l
-		core.repositoryformatversion=0
-		core.filemode=true
-		core.bare=false
-		core.logallrefupdates=true
-		remote.origin.url=https://github.com/moxiegirl/docker.git
-		remote.origin.fetch=+refs/heads/*:refs/remotes/origin/*
-		branch.master.remote=origin
-		branch.master.merge=refs/heads/master
-		user.name=Mary Anthony
-		user.email=mary@docker.com
-		remote.upstream.url=https://github.com/docker/docker.git
-		remote.upstream.fetch=+refs/heads/*:refs/remotes/upstream/*
+        $ git config --local -l
+        core.repositoryformatversion=0
+        core.filemode=true
+        core.bare=false
+        core.logallrefupdates=true
+        remote.origin.url=https://github.com/moxiegirl/docker.git
+        remote.origin.fetch=+refs/heads/*:refs/remotes/origin/*
+        branch.master.remote=origin
+        branch.master.merge=refs/heads/master
+        user.name=Mary Anthony
+        user.email=mary@docker.com
+        remote.upstream.url=https://github.com/docker/docker.git
+        remote.upstream.fetch=+refs/heads/*:refs/remotes/upstream/*
 
 	To list just the remotes use:
-		
-		$ git remote -v		
-		origin	https://github.com/moxiegirl/docker.git (fetch)
-		origin	https://github.com/moxiegirl/docker.git (push)
-		upstream	https://github.com/docker/docker.git (fetch)
-		upstream	https://github.com/docker/docker.git (push)
-	
-	
-		
+
```

Comment:
This change introduced a problem:

![image](https://cloud.githubusercontent.com/assets/1347209/6541785/874f74d0-c498-11e4-8561-65d52a4d5526.png)

Refer here to original to see: https://docs.docker.com/project/set-up-prereqs/#set-your-signature-and-an-upstream-remote the intentetion